src/averageAges.js

```diff
@@ -34,7 +43,22 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const filtred = people.filter(person => person.sex === 'f');
+  const searchMother = people.map(person => person.mother);
+
+  if (arguments.length !== 2) {
```

Comment:
```suggestion
  if (!withChildren) {
```
the same thing as in the previous function